spec/index.html

```diff
@@ -443,32 +529,50 @@ <h3>Triples</h3>
 
     <ul>
       <li>the <dfn>subject</dfn>, which is an
-      <a>IRI</a> or a <a>blank node</a></li>
+      <a>IRI</a>, a <a>blank node</a>, or a <a>quoted triple</a></li>
 
       <li>the <dfn>predicate</dfn>, which is an <a>IRI</a></li>
 
       <li>the <dfn>object</dfn>, which is an <a>IRI</a>,
-      a <a>literal</a> or a <a>blank node</a></li>
+      a <a>literal</a>, a <a>blank node</a>, or a <a>quoted triple</a></li>
     </ul>
 
     <p>An RDF triple is conventionally written in the order subject,
       predicate, object.</p>
 
+    <p>"RDF triple" is usually written "<a>triple</a>".<p>
+
+    <p>A <dfn data-local-lt="quoted">quoted triple</dfn> is an <a>RDF term</a> representing a <a>triple</a>
+      used as the <a>subject</a> or <a>object</a> of another triple.</p>
+
+    <p>An <dfn data-local-lt="asserted">asserted triple</dfn>
+      is an <a>RDF triple</a> that is an element of an <a>RDF graph</a>.</p>
+
     <p>The set of <span id="dfn-nodes"><!-- obsolete term--></span><dfn data-lt="node">nodes</dfn> of an <a>RDF graph</a>
-      is the set of subjects and objects of triples in the graph.
+      is the set of <a>subjects</a> and <a>objects</a> of <a>triples</a> in the graph
+      plus all the <a>nodes</a> of those <a>subjects</a> and/or its <a>objects</a>,
+      recursively, for <a>quoted triples</a>.
```

Comment:
Two things here:

1. This new way of defining the notion of "nodes of an RDF graph" depends on a notion of "nodes of a quoted triple" which is undefined at the moment. This issue can be fixed by defining this notion (see how the notion of constituent terms is defined in the CG report). However, I don't think that's even necessary because ...
3. ... I don't think we should change the definition of the notion of "nodes of an RDF graph" in this way. That's quite a substantial change. Are we sure that we don't break anything by this change? I mean, if this notion is used in other definitions somewhere in our documents, this change may affect these other definitions.

Reply:
Can you suggest a better alternative. Alternatively, we could add an issue marker that the consideration of nodes contained within quoted triples being considered as "nodes of an RDF graph" is open and subject to further deliberation by the group.

Reply:
I agree with @hartig  - we shouldn't change "nodes in an RDF graph". Section 1.1 is putting "graph" in general graph terminology context. We need to be careful because this is "node" in the sense of a graph-theory graph. nodes-arcs / vertex-edge.

I also would like to know if we need the terminology? "constituent" is used in the CG report in the mapping section, which is unlikely to be in RDF 1.2 and even if it is, is limited to semantics; and it is used in the "merge" text which is also not in concepts.

SPARQL uses general-graph nodes for property paths (specifically, "node set", which it defines).

Until proven to need terminology, it seems safer to not have i, allowing semantics to have in its scope.

Reply:
I also think that we should not change the definition of "node" at this point.